models/openai_model.py

```diff
@@ -719,7 +724,7 @@ async def send_summary_request(self, prompt, custom_api_key=None):
 
         tokens = self.usage_service.count_tokens(summary_request_text)
 
-        async with aiohttp.ClientSession(raise_for_status=True) as session:
+        async with aiohttp.ClientSession(raise_for_status=False) as session:

```

Comment:
Why did you make this not raise it anymore? Doesn't ValueError not catch other possible http exception cases? ValueError can only be possible if the API actually responds and we try to index the response json, but there are other client response errors that can occur right?

Reply:
I'm pretty sure their api responds properly in all the cases, the reason I changed it was that while I was playing with the models and max sizes and all I'd get 400 bad requests without any further information about what was bad in the request. So I changed it to catch the value error instead then retry on that because it actually gave the specific error from their api. I can change it back, I just wanted more verbosity